handsontable/src/plugins/contextMenu/menu/menu.js

```diff
@@ -626,6 +626,7 @@ export class Menu {
    * and adjusts the width and height of the menu's holder and hider elements accordingly.
    */
   updateMenuDimensions() {
+    const stylesHandler = this.hot.view.getStylesHandler();
```

Comment:
I think it's safer to use the menu instance here
```suggestion
    const stylesHandler = this.hotMenu.view.getStylesHandler();
```